tests/para-tests-no-ci/test-xcm-para.ts

```diff
@@ -282,7 +358,40 @@ describeParachain(
       expect(assetId).to.eq(assetIdB);
 
       // RELAYCHAIN
-      // send 1000 units to Baltathar in para A
+      // Set supported version
+      // Release-v0.9.12 does not have yet automatic versioning
+      await createBlockWithExtrinsicParachain(
```

Comment:
maybe we should add this to `registerAssetToParachain` if we're going to use it everytime

Reply:
registerAssetToParachain relates to something done in the parachain, while this is somethign to be done in the relay. I will create a separate function though